packages/kas/__tests__/kas_test.js

```diff
@@ -0,0 +1,85 @@
+import KAS from "../kas.js";
```

Comment:
Will adding `// @flow` cause a bunch of errors?

Reply:
Probably.  I'll try tackling flow and eslint in separate PRs.